charts/lagoon-remote/Chart.yaml

```diff
@@ -40,6 +40,8 @@ dependencies:
 # Valid supported kinds are added, changed, deprecated, removed, fixed and security
 annotations:
   artifacthub.io/changes: |
+    - kind: changed
+      description: update remote-controller to 0.33.0
```

Comment:
```suggestion
      description: update lagoon-build-deploy to 0.33.0
```

Reply:
@tobybellwood why would dependabot think this is `remote-controller`?

Reply:
😊 it was mEEEE